### DISCUSSION TITLE
Incomplete lap due to a technical issue

### DISCUSSION THREAD

**Discussion by telemetryf1** [theOehrly/Fast-F1#200] [Q&A] [answered]
Hi, I have a question for you. There is a way to get telemetry data for a lap that hasn't been completed due to a technical issue, e.g. Verstappen in Australia, or Perez in Canada, or Sainz in Baku? Thanks for your replies

**Comment by theOehrly** [comment, marked as answer]
Yes, the telemetry data for the full session and for each driver is contained in the dictionaries `Session.car_data` / `Session.pos_data` (https://theoehrly.github.io/Fast-F1/core.html#fastf1.core.Session.car_data)

Telemetry objects can be sliced using timestamps: https://theoehrly.github.io/Fast-F1/core.html#fastf1.core.Telemetry.slice_by_time

You can simply use the Time at which the last full lap was set as the first timestamp and add some amount of seconds to that for the second timestamp.

**Reply by telemetryf1** [reply]
Thanks so much
